components/tools/OmeroJava/test/integration/ManagedRepositoryTest.java

```diff
@@ -81,7 +81,7 @@
 @Test(groups = { "integration", "fs" })
 public class ManagedRepositoryTest extends AbstractServerImportTest {
     /* temporary file manager for sources of file uploads */
-    private static final TempFileManager tempFileManager = new TempFileManager(
+    protected static final TempFileManager tempFileManager = new TempFileManager(
```

Comment:
Maybe in the longer term we should move this up to `AbstractServerTest`.

Reply:
true but I don't want to "destabilize" anything now
Happy to follow that up post 5.4